src/app/feedback/feedback-form/feedback-form.component.ts

```diff
@@ -49,8 +50,9 @@ export class FeedbackFormComponent implements OnInit {
   }
 
   private getHeaderText(): string {
-    const el = this.el.element.nativeElement.querySelector('h1:not([style*="display:none"]');
-    return el ? el.innerHTML : '';
+    // const el = this.el.element.nativeElement.querySelector('h1:not([style*="display:none"]');
```

Comment:
When sending the feedback this is supposed to grab first visible h1 header and put it as a slide header to the feedback info.
Try using document.body instead of this.el.element.nativeElement, and see if the text is there when you send the feedback